libkbfs/keybase_daemon_rpc.go

```diff
@@ -538,6 +549,36 @@ func (k *KeybaseDaemonRPC) processUserPlusKeys(upk keybase1.UserPlusKeys) (
 	return u, nil
 }
 
+// LoadUnverifiedKeys implements the KeybaseDaemon interface for KeybaseDaemonRPC.
+func (k *KeybaseDaemonRPC) LoadUnverifiedKeys(ctx context.Context, uid keybase1.UID) (
+	[]VerifyingKey, []CryptPublicKey, error) {
+	cachedUserInfo := k.getCachedUserInfo(uid)
+	keys := len(cachedUserInfo.UnverifiedVerifyingKeys) +
+		len(cachedUserInfo.UnverifiedCryptPublicKeys)
+	if keys != 0 {
```

Comment:
Do you think it's worth distinguishing the case where we've already checked, and the user legitimately has 0 unverified keys?  For example, if this is a shared, finalized folder, but the user you're checking wasn't the one that reset their account.  Maybe that will never happen though, so this isn't a big deal?